### PR TITLE
Migrate to --ginkgo-parallel 

### PR DESCRIPTION
**Pull request by krzyzacy**
Let's clean up some more envs s/before/after/ set up 1.8 ci

### CODE REVIEW

**Comment by krzyzacy** [comment]
foooo
/joke
something-else

**Comment by krzyzacy** [comment]
Have time to do more cleanups now.

/unassign
/assign @zmerlynn @fejta @BenTheElder

File: jobs/config.json [hunk 1]
```diff
@@ -205,6 +205,7 @@
       "--extract=gci/gci-next-canary",
       "--gcp-project=e2e-cos-docker-val",
       "--gcp-zone=us-central1-f",
+      "--ginkgo-parallel=25",
```
Comment (BenTheElder) [inline jobs/config.json]: How did we get to 25 and 30?

**Comment by BenTheElder** [comment]
for tide :-)
/hold
feel free to cancel whenever @krzyzacy

File: jobs/env/ci-kubernetes-e2e-gce-alpha-api.env [hunk 2]
```diff
@@ -1,8 +1,6 @@
 ### job-env
-GINKGO_PARALLEL=y
 KUBE_OS_DISTRIBUTION=gci
 NUM_NODES=4
-GINKGO_PARALLEL_NODES=30
```
Comment (krzyzacy) [inline jobs/env/ci-kubernetes-e2e-gce-alpha-api.env]: @BenTheElder 30 comes from here

File: jobs/move_extract.py [hunk 3]
```diff
@@ -45,49 +45,39 @@ def sort():
...
+        parallism = False
+        parallism_nodes = 1
         for line in env.split('\n'):
             mat = regexp.search(line)
-            if not mat:
-                lines.append(line)
-                continue
-            if mat:  # These do nothing
+            if mat:
+                parallism_nodes = mat.group(1)
```
Comment (krzyzacy) [inline jobs/move_extract.py]: and this is the legacy migration script I've used

**Comment by krzyzacy** [comment]
(I'll merge this after we make sure pr-kops is working from my previous PR)

**Pull request merged**